src/task3/Complex.java

```diff
@@ -0,0 +1,61 @@
+package task3;
+
+public class Complex {
+    private int realNumber;
+    private int imaginaryNumber;
+
+    public Complex() {
+        this.realNumber = 0;
+        this.imaginaryNumber = 0;
+
+    }
+
+    public Complex(int realNumber, int imaginaryNumber) {
+        this.realNumber = realNumber;
+        this.imaginaryNumber = imaginaryNumber;
+    }
+
+    public int Re() {
+        return realNumber;
+    }
+
+    public int Im() {
+        return imaginaryNumber;
+    }
+
+
+    public Complex add(Complex z1) {
+        int realNumber = this.realNumber + z1.realNumber;
+        int imaginaryNumber = this.imaginaryNumber + z1.imaginaryNumber;
+        return new Complex(realNumber, imaginaryNumber);
```

Comment:
no need for additional variables,

you could have simply `return new Complex(this.realNumber + z1.realNumber, this.imaginaryNumber + z1.imaginaryNumber)`

Also, while this works it would be advisable to use `z1.Re() `and `z1.Im()` for the parameter. This would emphasise the difference between current object `this` and `z1`, and also if `z1` wasn't of type `Complex` but some other type instead you would not be able to access it's attributes here, but we will learn this later on.

`z1` could have been named `z`.